packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -38,28 +46,49 @@ export async function openTransport(device: USBDevice): Promise<TransportWebUSB>
   }
 }
 
-export async function getTransport(): Promise<TransportWebUSB> {
-  if (!(window && window.navigator.usb)) throw new core.WebUSBNotAvailable();
+export const getLedgerTransport = async (): Promise<TransportWebUSB> => {
+  // TODO(gomes): move all the following implementation to an actual sane getTransport
+  const device = await getFirstLedgerDevice();
+
+  if (!device) throw new Error("No device found");
+
+  await device.open();
+  if (device.configuration === null) await device.selectConfiguration(1);
 
   try {
-    return (await TransportWebUSB.request()) as TransportWebUSB;
+    await device.reset();
   } catch (err) {
-    if (core.isIndexable(err) && err.name === "TransportInterfaceNotAvailable") {
-      throw new core.ConflictingApp("Ledger");
-    }
+    console.warn(err);
+  }
+
+  const inyerface = device.configurations[0].interfaces.find(({ alternates }) =>
```

Comment:
That's fun and all but not very greppable
```suggestion
  const interface = device.configurations[0].interfaces.find(({ alternates }) =>
```